### PR TITLE
Update config for v1.1.2

### PR DESCRIPTION
**Pull request by andreibancioiu**
Update configuration with respect to Node version `v1.1.2` (pre-release).

### CODE REVIEW

**Review by LucianMincu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: systemSmartContractsConfig.toml [hunk 1]
```diff
@@ -3,8 +3,8 @@
     MinStakeValue = "100000000000000000000" #100 eGLD
     UnBondPeriod = 144000
     MinStepValue = "100000000000000000000"
-    AuctionEnableNonce = 100000000000
-    StakeEnableNonce = 100000000000
+    AuctionEnableEpoch = 100000000000
```
Comment (SebastianMarian) [inline systemSmartContractsConfig.toml:6]: If epoch is uint32 then this value is overflowed and its value is unpredictable. You can use the same value used above to SwitchJailWaitingEnableEpoch = 1000000

File: systemSmartContractsConfig.toml [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+    StakeEnableEpoch = 100000000000
```
Comment (SebastianMarian) [inline systemSmartContractsConfig.toml:7]: The same here

File: systemSmartContractsConfig.toml [hunk 3]
```diff
@@ -16,12 +16,12 @@
 [ESDTSystemSCConfig]
     BaseIssuingCost = "5000000000000000000" #5 eGLD
     OwnerAddress = "erd1ze6pgefwc7mdx0rhje3h50kkgt26ut3gx8ytezndkz8fjcf96gsqsd7ugw"
-    Disabled = true
+    EnabledEpoch = 100000000000
```
Comment (SebastianMarian) [inline systemSmartContractsConfig.toml:19]: The same here

File: systemSmartContractsConfig.toml [hunk 4]
```diff
@@ -16,12 +16,12 @@
...
-	MinQuorum    = 400
-	MinPassThreshold = 300
-	MinVetoThreshold = 50
-	Disabled     = true
+    ProposalCost = "5000000000000000000" #5 eGLD
+    NumNodes     = 500
+    MinQuorum    = 400
+    MinPassThreshold = 300
+    MinVetoThreshold = 50
+    EnabledEpoch = 100000000000
```
Comment (SebastianMarian) [inline systemSmartContractsConfig.toml:27]: The same here

**Pull request merged**